app/Jobs/CheckTraefikVersionForServerJob.php

```
@@ -25,7 +25,9 @@ class CheckTraefikVersionForServerJob implements ShouldBeEncrypted, ShouldQueue
      */
     public function __construct(
         public Server $server,
-        public array $traefikVersions
+        public array $traefikVersions,
+        public bool $shouldNotify = true,
```

PHPMD (2.15.0)

[error] The method __construct has a boolean flag argument $shouldNotify, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)